src/backend_tools/db_provider.ts

```diff
@@ -1,39 +1,36 @@
 import {
   doc,
-  Firestore,
+  type Firestore,
   getFirestore,
   setDoc,
   updateDoc,
   addDoc,
   getDoc,
-  DocumentData,
-  Query,
-  where,
+  type DocumentData,
   getDocs,
-  QueryDocumentSnapshot,
-  query,
-  QuerySnapshot,
   deleteDoc,
   collection,
   arrayUnion,
+  query,
 } from 'firebase/firestore';
-import Club from '../models/club';
-import User from '../models/user';
+import type Club from '../models/club';
+import type User from '../models/user';
 import FirebaseApp from './firebase';
-import Event from '../models/event';
+import type Event from '../models/event';
+import Fuse from 'fuse.js';
 
 class DbProvider {
   db: Firestore = getFirestore(FirebaseApp);
-  userPath: string = 'Users';
```

Comment:
What's the reason to remove the type annotation?

Reply:
It automatically type annotates it under the hood so it's just not necessary